### ISSUE TITLE
New videos for Jakarta EE 9.1 

### ISSUE THREAD

**Issue by cesarhernandezgt** [jakartaee/jakarta.ee#1172]
During today's Marketing Committee call, we identified the need of having new videos for the Jakarta EE 9.1 release page.
This was also visualized in a comment from [issue1165](https://github.com/jakartaee/jakarta.ee/issues/1165#issuecomment-840178200)

So far the two videos that will need to be edited and created are:

1. Create a video base on the Jakarta EE 9.1 Update Kevin Sutter provided during the last Jakarta EE Update Call | April (minute 2:24 - 12:00)

2. Create a new video highlighting Jakarta EE 9.1 by  Ivar Grimstad Jakarta Developer Advocate.

**Comment by TanjaObradovic** [comment]
Thanks Cesar! I have assigned it to @karenmcnaughton.

**Comment by karenmcnaughton** [comment]
I have looped in @ivargrimstad to help with producing a video for 2. above ^

**Comment by cesarhernandezgt** [comment]
First video is now available https://github.com/jakartaee/jakarta.ee/issues/1165#issuecomment-845988961

**Comment by karenmcnaughton** [comment]
Hello @cesarhernandezgt are you taking care of getting this on the page as well: Create a video base on the Jakarta EE 9.1 Update Kevin Sutter provided during the last Jakarta EE Update Call | April (minute 2:24 - 12:00)

I understood you had this action? Can you help pls.

**Comment by cesarhernandezgt** [comment]
@karenmcnaughton 

I added to the PR the defined interval for the second video via https://github.com/jakartaee/jakarta.ee/pull/1170/commits/21e43681db7d5119536c9a7658dbbdc160892a34

Live preview: https://deploy-preview-1170--jakartaee.netlify.app/release/9.1

Notice that for 9.1 videos, I increased the size of the video a bit. I discovered that for Jakarta 9 release page, a user couldn't maximize any of the four videos to full screen. I opened https://github.com/jakartaee/jakarta.ee/issues/1191 so your team can follow up. 

After your review, if it applies, feel free to close this GitHub issue.
